slack-api-model/src/main/java/com/slack/api/model/block/element/RichTextSectionElement.java

```diff
@@ -146,7 +150,23 @@ public static class TextStyle {
         private boolean bold;
         private boolean italic;
         private boolean strike;
+        private boolean highlight;
+        private boolean clientHighlight;
+        private boolean unlink;
         private boolean code;
     }
 
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class NoCodeTextStyle {
```

Comment:
Added this for the ones that do not accept code

Reply:
It seems like this object may be referenced as `limited_style`, what do you think about naming this `LimitedTextStyle`?

```suggestion
    public static class LimitedTextStyle {
```

Reply:
Sounds good. Renamed it.